src/templates/experiments-status.html

```
@@ -6,9 +6,9 @@
   'experiments_status',
   'experiments_status',
   'Create and modify the status available to experiments.'|trans,
-  'Experiments status can be associated with an Experiment or an Experiment Template as a Status. They belong to a team and have two main properties: name and color. Examples: Running, Done, Need to be redone, Failed, etc...'|trans,
+  'Experiment status can be associated with an Experiment or an Experiment Template as a Status. They belong to a team and have two main properties: name and color. Examples: Running, Done, Need to be redone, Failed, etc...'|trans,
   statusArr|filter(c => c.is_current_team == 1),
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)